services/workflows-service/src/webhooks/webhooks.service.ts

```diff
@@ -68,8 +74,8 @@ export class WebhooksService {
             ...rest,
             additionalInfo: rest.additionalInfo ?? {},
           },
-          id: entityId,
-          ballerineEntityId: entityId,
+          id: endUserId,
```

Comment:
when creating a workflow, ID is mapped to correlationId (the id in the user's database) - so I'm not sure about this line - lets talk about it